docs/ref/applications.txt

```diff
@@ -41,6 +41,22 @@ interact with installed applications, mainly for configuration and also for
 introspection. That's why the application registry maintains metadata in an
 :class:`~django.apps.AppConfig` instance for each installed application.
 
+Here is a comparison of "Project vs Application" for better understanding:
+
++--------------------+--------------------+------------------------------+
+| Topic              | Project            |Application                   |
++====================+====================+==============================+
+|Contains            | Settings           | Code, Views, Templates, ...  |
++--------------------+--------------------+------------------------------+
+|Size                | Only view lines    | Can be huge                  |
++--------------------+--------------------+------------------------------+
+|Can have Models     | No                 | Yes                          |
```

Comment:
I don't think there's a restriction preventing a project from also being an application that has models.